third_party/move/tools/move-linter/src/model_ast_lints/assert_const.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module implements an expression linter that checks for assert!()s
+//! where the condition is either `true` or `false`.
+//! Note: As a side-effect, the linter also checks if blocks that are
```

Comment:
Module documentation should not refer to tests for explanation of behavior (we could end up changing test organization or do other refactorings). We can easily provide an actual example here.